csrankings-m.csv

```diff
@@ -1921,7 +1923,7 @@ Min Wu 0001,University of Maryland - College Park,https://www.umiacs.umd.edu/peo
 Min Xian,University of Idaho,http://www.uidaho.edu/engr/our-people/min-xian,NOSCHOLARPAGE
 Min Yang 0002,Fudan University,http://www.cs.fudan.edu.cn/?page_id=2665###,UnKf9FIAAAAJ
 Min Yao,Zhejiang University,http://mypage.zju.edu.cn/myao,NOSCHOLARPAGE
-Min Zhang 0005,Soochow University,https://zhangminsuda.github.io/Pesonel-Information,CncXH-YAAAAJ
+Min Zhang 0005,Harbin Institute of Technology,https://zhangminsuda.github.io/Pesonel-Information,CncXH-YAAAAJ

```

Comment:
At Soochow, adjunct only at Harbin per home page.

Reply:
Dear emeryberger,Min Zhang 0005 will be the full-time PHD superviser of HITsz(Harbin Institute of Technology, ShenZhen campus). Because his Chinese Homepage shows that his working experience in Soochow University ended in 2021.9. I think that he may not update his English Homepage because he has not finished his demission procedure. And some Chinese news shows that he is "supervising students in HITsz" and " becoming the assistant of the president of HITsz".

If you still think this modification for him is not "safe", I hope you can merge this PR firstly. And then I will email Min Zhang 0005 or his students, waiting for them to rectify this modification personaly. Because this PR still contains other important modifications for other faculties. And I always meet some odd errors, when I change the PR.

Reply:
Thanks for the detailed explanation - this is fine.